### PR TITLE
filter out the price data that not have enough precision.

### PR DESCRIPTION
**Pull request by btsabc**
Signed-off-by: kinglaw <58291@qq.com>

### CODE REVIEW

File: libraries/plugins/market_history/market_history_plugin.cpp [hunk 1]
```diff
@@ -174,17 +174,25 @@ struct operation_process_fill_order
              db.modify( *itr, [&]( bucket_object& b ){
                   b.base_volume += trade_price.base.amount;
                   b.quote_volume += trade_price.quote.amount;
-                  b.close_base = trade_price.base.amount;
-                  b.close_quote = trade_price.quote.amount;
-                  if( b.high() < trade_price ) 
+                  if(trade_price.base.amount >= 100 && trade_price.quote.amount >= 100)
```
Comment (abitmore) [inline libraries/plugins/market_history/market_history_plugin.cpp]: Although this is okay for most "popular" trading pairs, it's not ideal to be applied to all pairs, especially UIA's.

I think it's better that we record the data that how much difference is caused by rounding when matching orders, or simply record the "fair" price there, then process the data here.

**Comment by landry314** [comment]
Is this related to https://github.com/bitshares/bitshares-ui/issues/200?

**Comment by pmconrad** [comment]
The intent of this PR is unclear, but I think it is likely an attempt to fix that exact problem.

**Comment by landry314** [comment]
@abitmore, is your PR related to bitshares/bitshares-ui#200?

I am considering opening a new issue in the core for this because @svk31 thinks the issue goes deeper than the GUI and must be fixed in the core. He explains it here: https://github.com/bitshares/bitshares-ui/issues/200#issuecomment-337844372.

I don't want to create a duplicate issue, so I ask here first, but I am happy to open a new issue and link to the past discussion.

The charts (long wicks mainly) and estimated account value are screwed up by these "fractional dust" trades executing out of range... it is a serious issue because we need good charts to get people to stay on the DEX.

**Comment by abitmore** [comment]
@landry314 it's not "my" PR. Please go ahead and create an issue. Better to discuss there.

**Comment by abitmore** [comment]
Please discuss in #449.

**Comment by abitmore** [comment]
Superseded by #455, closing this one.

**Pull request closed without merging**